docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+- [Temporal Go SDK](/develop/go/nexus) support for Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+- [Temporal Java SDK](/develop/java/nexus) support for Nexus is available in [Pre-release](/evaluate/development-production-features/release-stages#pre-release).
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+Nexus allows you to reliably connect your Temporal Applications.
+It was designed with Durable Execution in mind and enables each team to have their own Namespace for improved modularity, security, troubleshooting, and fault isolation.
+
+Nexus Services can be used within and across Namespaces, which are connected by a global mTLS-secured Envoy mesh in Temporal Cloud.
+Nexus Operations are arbitrary-duration operations (sync or async) that can be implemented with Temporal primitives, like Workflows, or execute arbitrary code.
+
+Unlike other forms of inter-service communication, Nexus combines a familiar programming model with the resiliency of the Temporal Platform and it's queue-based Worker architecture.
+The result is an integrated Temporal experience with improved security, observability, and reliability for cross-team, cross-domain, cross-namespace, and multi-region use cases.
+
+**Why use Temporal Nexus?**
+
+Temporal Nexus provides:
+
+- **A fully integrated Temporal SDK experience**:
+  - Use Nexus Services from a caller Workflow.
+  - Run Nexus Services in a Worker, often the same Worker as underlying primitives.
+  - Implement long-running asynchronous Nexus Operations as Workflows.
+  - Handle low-latency synchronous Nexus Operations with Temporal primitives or arbitrary code.
+  - At-least-once execution. Option for at-most-once execution using handler Workflow policy.
+- **Nexus Endpoints with a queue-based Worker architecture**:
+  - Connect callers and handlers through Nexus Endpoints, a reverse proxy for Nexus Services.
+  - Manage Endpoints in the Temporal Nexus Registry.
+  - Use an Nexus Endpoint by name, which routes to an upstream target Namespace and Task Queue.
+  - Workers poll an Endpoint's target Task Queue, for automatic load balancing.
+- **Built-in Temporal Nexus Machinery**:
+  - Reliable state-machine-based invocation and completion callbacks for Nexus Operations.
```

Comment:
Now these are all nouns/features. It feels like a shift in tone.

Reply:
Yes, I ran out of verbs here.

Will split into 2 sections:

Integrated Temporal experience:
- verb
- verb

Key features:
- noun
- noun